brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3482 Fix: Container - Selecting Video as background and adding any block on it flex properties not getting applied.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3482

### Screenshots

- https://d.pr/v/BMPscE

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to master
- Drag and drop container.
- Set Content Width full.
- Minimum Height to 500 and then set Content width to Full width.
- Then drag and drop CTA inside container with Stack on desktop and overall alignment center.
- Publish the page and visit frontend. You will see gap between Read More button and CTA description.
- To fix this issue let's switch to branch and regenerate the assets and refresh the page issue will get resolved.

### Checklist:
